src/components/MyAccount/Settings/UsernameForm.tsx

```diff
@@ -151,21 +162,35 @@ const UsernameForm = ({ patron, usernameState }: UsernameFormProps) => {
     <Flex alignItems="center" marginTop={{ base: "unset", md: "-xs" }}>
       {usernameInSierra ? (
         <>
-          <Text size="body1" sx={{ marginBottom: 0 }}>
+          <Text
+            size="body1"
+            sx={{ marginBottom: 0, width: { base: "l", sm: "250px" } }}
+          >
             {usernameInSierra}
           </Text>
           <EditButton
+            ref={focusRef as React.Ref<HTMLButtonElement>}
```

Comment:
From the comment in `PasswordForm` regarding breaking up the ref into two refs, doing so will allow this `as React.Ref...` to be removed.